### DISCUSSION TITLE
No folder 'primevue/styled' in node_modules

### DISCUSSION THREAD

**Discussion by JensDeMuynck** [primefaces/community#1976] [PrimeVue] [answered]
I tried installing primevue@4.0.0-beta.3 in a Laravel/Inertia/Vue app, but got the error `Failed to resolve import "primevue/styled" from "resources/js/app.js". Does the file exist?`

I looked inside the node_modules/primevue folder, and could indeed not find a folder called 'styled'.

The folder 'config' does exist, so importing the unstyled version does work, but that's not what i want.

**Comment by tugcekucukoglu** [comment, marked as answer]
It's a deprecated feature. For the latest updates: https://v4.primevue.org/configuration/

**Reply by JensDeMuynck** [reply]
I see, so [this page](https://v4.primevue.org/vite/) should be updated then?
